applications/klighd-cli/src/main.ts

```diff
@@ -22,11 +22,11 @@ import {
     createKlighdDiagramContainer,
     requestModel,
     getActionDispatcher,
-    SetPreferencesAction,
     bindServices,
     SetInitialBookmark,
     Bookmark,
 } from "@kieler/klighd-core";
+import { SetPreferencesAction } from "@kieler/klighd-core/lib/options/actions"
```

Comment:
For the API this should be importable as before from `@kieler/klighd-core` without the `lib/...` part after it.